src/components/Reports/ReportList.tsx

```diff
@@ -1,6 +1,63 @@
-import React from 'react';
+import { Container, Grid, Typography, useTheme } from '@mui/material';
+import { TableColumnType } from '@terraware/web-components';
+import Table from 'src/components/common/table';
+import React, { useEffect, useRef, useState } from 'react';
 import TfMain from 'src/components/common/TfMain';
+import ReportService from 'src/services/ReportService';
+import strings from 'src/strings';
+import { ListReport } from 'src/types/Report';
+import { getRequestId, setRequestId } from 'src/utils/requestsId';
+import PageHeaderWrapper from '../common/PageHeaderWrapper';
+import ReportsCellRenderer from './TableCellRenderer';
+import { useOrganization } from 'src/providers';
 
 export default function ReportList(): JSX.Element {
-  return <TfMain>Report List Page</TfMain>;
+  const contentRef = useRef(null);
+  const [results, setResults] = useState<ListReport[]>([]);
+  const { selectedOrganization } = useOrganization();
+  const theme = useTheme();
+
+  const columns: TableColumnType[] = [
+    { key: 'name', name: strings.REPORT, type: 'string' },
+    { key: 'status', name: strings.STATUS, type: 'string' },
+    { key: 'modifiedByName', name: strings.LAST_EDITED_BY, type: 'string' },
+    { key: 'submittedByName', name: strings.SUBMITTED_BY, type: 'string' },
+    { key: 'submittedTime', name: strings.DATE_SUBMITTED, type: 'string' },
+  ];
+
+  useEffect(() => {
+    const refreshSearch = async () => {
+      const requestId = Math.random().toString();
+      setRequestId('searchReports', requestId);
+
+      const reportsResults = await ReportService.getReports(selectedOrganization.id);
+
+      if (getRequestId('searchReports') === requestId) {
+        setResults(reportsResults.reports || []);
+      }
+    };
+
+    refreshSearch();
+  }, [selectedOrganization.id]);
+
+  return (
+    <TfMain>
+      <Grid container>
+        <PageHeaderWrapper nextElement={contentRef.current}>
+          <Grid item xs={12} sx={{ paddingLeft: 3, marginBottom: 4 }}>
+            <Typography sx={{ fontSize: '24px', fontWeight: 600 }}>{strings.REPORTS}</Typography>
+          </Grid>
+        </PageHeaderWrapper>
+        <Container
+          ref={contentRef}
+          maxWidth={false}
+          sx={{ padding: 3, borderRadius: 4, backgroundColor: theme.palette.TwClrBaseWhite }}
+        >
+          <Grid item xs={12}>
+            <Table id='reports-table' columns={columns} rows={results} orderBy='name' Renderer={ReportsCellRenderer} />
```

Comment:
use pre-sorted option?

Reply:
This is not using the search api, it is using a reports specific api

Reply:
Oops.. ok good point. 
Realized we don't use [locale compare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) for strings in the table comparator. For spanish we should be ok? I can create a ticket to pass locale to table..

Reply:
In the example it is comparing two strings in different locales. Table has same locale for all its rows, so why is this necessary?

Reply:
The example may be doing that.. locale compare uses language specific rules for comparison. Rules vary by language.
Uses Intl.Collator behind the scenes.. see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator

Reply:
Didn't mean to add any distraction.. not requesting any changes here right now.. just brought that up for discussion. We can discuss this offline with the team maybe?

Reply:
Ok! I see now, yes, we may need it